.changeset/odd-scissors-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/instruxi-sxt-adapter': major
+---
+
+Rebase and redo a PR
```

Comment:
Please change this to something like 
"Create new instruxi-sxt-adapter"

Reply:
Should i create a new changeset or just change the contents of the file and push to the branch

Reply:
Simply change the content of this file